src/client/src/utils/constants.ts

```diff
@@ -11,6 +11,11 @@ const PROJECT_NAME_CHARACTER_LIMIT = 50;
 
 const MAX_PAGES_ALLOWED = 20;
 
+const WEB_TEMPLATE_STUDIO = {
```

Comment:
NIT: Maybe WEB_TEMPLATE_STUDIO_LINKS name would be more clear in imports?

Reply:
Thanks. Updated